I'm sorry, but errors were detected in your script. Please correct the
errors listed below, and try again.


The label _call_prepare_tutorial_battle is defined twice, at File "game/weeks/week_1/day_1.rpy", line 109:
            call prepare_tutorial_battle from _call_prepare_tutorial_battle
and File "game/weeks/week_3/day_19.rpy", line 287:
    call prepare_tutorial_battle from _call_prepare_tutorial_battle


Ren'Py Version: Ren'Py 8.1.3.23091805
Sun Oct 29 11:23:09 2023
